Failed to launch on aarch64-linux:

```
$ result/bin/briar-desktop
04:59:35.264 [main] INFO  org.briarproject.briar.desktop.Main - This is briar-desktop version 0.4.0-beta
04:59:35.267 [main] INFO  org.briarproject.briar.desktop.Main - Build info:
04:59:35.268 [main] INFO  org.briarproject.briar.desktop.Main -   Git hash 1a1028b6afd2567daee5d04cd116ac58036ab49d
04:59:35.269 [main] INFO  org.briarproject.briar.desktop.Main -   Commit time 2023-01-31 15:23:23
04:59:35.270 [main] INFO  org.briarproject.briar.desktop.Main -   Tag 0.4.0-beta
04:59:35.906 [main] WARN  org.briarproject.briar.desktop.Main - Uncaught exception in thread main
java.lang.ExceptionInInitializerError: null
        at ...
Caused by: org.jetbrains.skiko.LibraryLoadException: Cannot find libskiko-linux-arm64.so.sha256, proper native dependency missing.
        at org.jetbrains.skiko.Library.findAndLoad(Library.kt:105)
        at org.jetbrains.skiko.Library.load(Library.kt:59)
        at org.jetbrains.skiko.SkiaLayer.<clinit>(SkiaLayer.awt.kt:28)
        ... 32 common frames omitted
```

--- a/pkgs/applications/networking/instant-messengers/briar-desktop/default.nix
+++ b/pkgs/applications/networking/instant-messengers/briar-desktop/default.nix
@@ -19,11 +19,11 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "briar-desktop";
-  version = "0.3.1-beta";
+  version = "0.4.0-beta";
 
   src = fetchurl {
-    url = "https://desktop.briarproject.org/jars/linux/0.3.1-beta/briar-desktop-linux-0.3.1-beta.jar";
-    sha256 = "841dc198101e6e8aa6b5ab6bd6b80e9c6b2593cb88bc3b2592f947baf963389d";
+    url = "https://desktop.briarproject.org/jars/linux/${version}/briar-desktop-linux-${version}.jar";
+    hash = "sha256-7zeIWsdPvROHGaf5igodlZss6Gow3kp6PX+QAzmxMnw=";
   };
 
   dontUnpack = true;
@@ -58,6 +58,6 @@ stdenv.mkDerivation rec {
     homepage = "https://code.briarproject.org/briar/briar-desktop";
     license = licenses.gpl3;
     maintainers = with maintainers; [ onny ];
-    platforms = [ "x86_64-linux" "aarch64-linux" "armv7l-linux" ];
+    platforms = [ "x86_64-linux" ];
   };
 }

